Packs/CommonPlaybooks/Playbooks/playbook-Extract_Indicators_From_File_-_Generic_v2_4_5.yml

```diff
@@ -601,23 +560,32 @@ tasks:
             - operator: containsString
               left:
                 value:
-                  simple: File.Info
+                  simple: inputs.File.Info
                 iscontext: true
               right:
                 value:
                   simple: pdf
               ignorecase: true
           accessor: EntryID
           transformers:
+          - operator: AppendIfNotEmpty
+            args:
+              item:
+                value:
+                  simple: ${File(val.Info=="application/pdf").EntryID}
```

Comment:
should we compare against `inputs.File`?

Reply:
This should be `File`, because we are taking the `inputs.File`s that are PDFs, and then appending the `File`s converted into PDFs (the `File` is the output of the conversion command)